test/inductor/test_inductor_freezing.py

```diff
@@ -280,21 +280,28 @@ def foo(mod, x):
             self.assertEqual(out_compiled_no_inference, out_compiled)
 
     def test_folded_conv_bn(self):
-        for use_bias, dtype in itertools.product(
-            [True, False], [torch.float16, torch.bfloat16, torch.float32]
+        class Model(torch.nn.Module):
+            def __init__(self, has_activation):
+                super().__init__()
+                self.conv_bn = ConvBN(3, 32, bias=use_bias, kernel_size=3, stride=2)
+                self.relu6 = torch.nn.ReLU6()
+                self.has_activation = has_activation
+
+            def forward(self, x):
+                if self.has_activation:
+                    return self.relu6(self.conv_bn(x))
+                return self.conv_bn(x)
+
+        for use_bias, dtype, has_activation in itertools.product(
+            [True, False], [torch.float16, torch.bfloat16, torch.float32], [True, False]
```

Comment:
could we just register an additional iter of `use_bias=True, dtype=torch.float16`, `has_activation=True` ? 
dont want this test to run for too long

Reply:
updated.